Source/SwiftLintCore/Rules/CustomRules.swift

```diff
@@ -79,19 +83,27 @@ struct CustomRules: Rule, CacheDescriptionProvider {
                                severity: configuration.severity,
                                location: Location(file: file, characterOffset: $0.location),
                                reason: configuration.message)
-            }).filter { violation in
-                guard let region = file.regions().first(where: { $0.contains(violation.location) }) else {
```

Comment:
Where is this checked now?

Reply:
Everything now happens in `superfluousDisableCommandViolations` - I think this was the root of the problem with the previous PR.